examples/serialization/load.py

```diff
@@ -0,0 +1,27 @@
+import chainer
+import numpy as np
+import model
+
+
+n_in = 784
+n_units = 100
+n_out = 10
+
+# Create model object first
```

Comment:
Logical separation of this script is not clear at first glance.
How about separating into functions, like `load_npz_file` and `load_hdf5_file` (or any better names)?

ditto. for `save` script.